website/docs/reference/appsmith-framework/widget-actions/post-message.md

```diff
@@ -21,6 +23,57 @@ postWindowMessage(message, targetIframe, targetOrigin)
 
 To see examples of the **Post message** action, take a look at the [sample app](https://app.appsmith.com/applications/61f3d1949d6d6a6720c98681/pages/61f3d1949d6d6a6720c98684).
 
+## Parent message
```

Comment:
This can be used in the example scenario to show case how the communication happens from the parent app to Appsmith.

Reply:
Agree. we can remove this heading altogether